test/controllers/EventReportControllerSpec.scala

```diff
@@ -229,17 +212,36 @@ class EventReportControllerSpec extends AsyncWordSpec with Matchers with Mockito
     }
   }
 
-
   "getEvent" must {
-    "return OK with the json response" in {
+    "return OK for Event 1 with dummy json response" in {
+      when(mockEventReportService.getEvent(
+        ArgumentMatchers.eq(pstr),
+        ArgumentMatchers.eq(startDate),
+        ArgumentMatchers.eq(versionString),
+        ArgumentMatchers.eq(Event1)
+      )(any(), any()))
+        .thenReturn(Future.successful(dummyJsValue))
+
+      val result = controller.getEvent(fakeRequest.withHeaders(
+        newHeaders = "pstr" -> pstr,
+        "startDate" -> startDate,
+        "version" -> "1",
+        "eventType" -> Event1.toString
+      ))
+
+      status(result) mustBe OK
+      contentAsJson(result) mustBe dummyJsValue
+    }
+
+    "return OK for Event 2 with dummy json response" in {
       when(mockEventReportService.getEvent(
         ArgumentMatchers.eq(pstr),
         ArgumentMatchers.eq(startDate),
         ArgumentMatchers.eq(versionString),
         ArgumentMatchers.eq(Event2)
       )(any(), any()))
         .thenReturn(Future.successful(dummyJsValue))
-      val controller = application.injector.instanceOf[EventReportController]
+
```

Comment:
we can add one more test here - when we pass invalid eventType, it should throw `new BadRequestException(s"Bad Request: invalid eventType ($eventType)")` .